data/ecosystems/s/solana.toml

```diff
@@ -18201,3 +18201,1933 @@ url = "https://github.com/ZYJLiu/versioned-transaction"
+
+[[repo]]
+url = "https://github.com/Aurora0116/mcn-frontend-nextjs-graphql-master"
+
+[[repo]]
+url = "https://github.com/catchspider2002/grizzlython-hackathon"
+
+[[repo]]
+url = "https://github.com/okapiHQ/okapi-api-grizzlython-submission"
+
+[[repo]]
+url = "https://github.com/mohandesjiri/solana-tx-size"
+
+[[repo]]
+url = "https://github.com/ecoWarriors/web-ecotoken"
+
+[[repo]]
+url = "https://github.com/zillerium/encodesolana"
+
+[[repo]]
+url = "https://github.com/nico1aseth/mine-builder"
+
+[[repo]]
+url = "https://github.com/Ivan-hox/metadata"
+
+[[repo]]
+url = "https://github.com/qudo-code/template--turborepo"
+
+[[repo]]
+url = "https://github.com/CompendiumFi/Compendex-Openbook-UI"
+
+[[repo]]
+url = "https://github.com/Smokefarmer/NFT_Examples"
+
+[[repo]]
+url = "https://github.com/Rylanqwe/please"
+
+[[repo]]
+url = "https://github.com/Ace82/Web3-Solana-Project"
+
+[[repo]]
+url = "https://github.com/brknarsy/metaplex-token-minter"
+
+[[repo]]
+url = "https://github.com/0xnetfox/rb-access"
+
+[[repo]]
+url = "https://github.com/kizmt/myepicproject"
+
+[[repo]]
+url = "https://github.com/hardiktheprogrammer/Aadaam-"
+
+[[repo]]
+url = "https://github.com/N044H/Tiny-Crew-Mint"
+
+[[repo]]
+url = "https://github.com/un-focused/VybeNetworksInterview"
+
+[[repo]]
+url = "https://github.com/Fineas/ISTM-List"
+
+[[repo]]
+url = "https://github.com/billyzhang663/solana-dapp-post-feed"
+
+[[repo]]
+url = "https://github.com/ilovespectra/heliumdenver"
+
+[[repo]]
+url = "https://github.com/0xPratik/web3Social-Gum"
+
+[[repo]]
+url = "https://github.com/thebobfather2/funziessite"
+
+[[repo]]
+url = "https://github.com/h4rkl/solana-mui-dapp-scaffold"
+
+[[repo]]
+url = "https://github.com/diegosano/web3dev-solana-gif-portal"
+
+[[repo]]
+url = "https://github.com/onuraladi/arac"
+
+[[repo]]
+url = "https://github.com/Manishsahusahu/social-media-3.0"
+
+[[repo]]
+url = "https://github.com/MundoKaizen/Ackee_Final"
+
+[[repo]]
+url = "https://github.com/harshmathurx/candy_machine"
+
+[[repo]]
+url = "https://github.com/sumitchoudharySEM/candy_machine"
+
+[[repo]]
+url = "https://github.com/theFutureGuy/NFTsolana"
+
+[[repo]]
+url = "https://github.com/simulacra10/rothbard-token"
+
+[[repo]]
+url = "https://github.com/gr3y6h0st/playwright"
+
+[[repo]]
+url = "https://github.com/ghanatava/sprinthacks"
+
+[[repo]]
+url = "https://github.com/matr-world/get-mintlist-ui-sveltkit"
+
+[[repo]]
+url = "https://github.com/Domino-Blockchain/sample-domi-contracts"
+
+[[repo]]
+url = "https://github.com/BFF-codelet/bff-codelet"
+
+[[repo]]
+url = "https://github.com/formfunction-hq/formfunction-program-shared"
+
+[[repo]]
+url = "https://github.com/bechir-fahmi/DynoExtension"
+
+[[repo]]
+url = "https://github.com/catdev0/goldbox"
+
+[[repo]]
+url = "https://github.com/gakker/automation-frontend"
+
+[[repo]]
+url = "https://github.com/cubik-so/cubik"
+
+[[repo]]
+url = "https://github.com/theia-ailabs/theia-client"
```

Comment:
Not a big deal since it's only 2 contributors, but it definitely uses Solana https://github.com/theia-ailabs/theia-client/blob/main/app/package.json#L22